cpu/msp430_common/Kconfig

```
@@ -6,6 +6,27 @@
 # directory for more details.
 #
 
+config MODULE_MSP430_COMMON
+    bool
+    default y if CPU_CORE_MSP430
+    depends on TEST_KCONFIG
+    imply MODULE_NEWLIB_NANO
+    select MODULE_MALLOC_THREAD_SAFE if TEST_KCONFIG
```

As this already depends on `TEST_KCONFIG`, we could:
```suggestion
    select MODULE_MALLOC_THREAD_SAFE
```

---

Seems like the if is actually needed I'll revert that commit, see https://ci.riot-os.org/RIOT-OS/RIOT/17290/0c160ff81bfb2b06567a6b87bd56538637356995/output/compile/tests/gnrc_sixlowpan_frag_minfwd/msb-430h:gnu.txt

---

Hmm according to the error, it is not this symbol that was causing the invalid `select`, but `CPU_ARCH_MSP430`.

---

Ahh thanks